src/components/App/App.tsx

```diff
@@ -73,9 +77,14 @@ const App = () => {
       <Suspense fallback={<Loader isPage={true} />}>
         <Switch>
           <Route path="/all">{returnComponent('all')}</Route>
+          <Route path="/voucher/:id">
+            <VoucherProvider>
+              <VoucherRedemptionPage />
+            </VoucherProvider>
+          </Route>
           <Route path="/merchants">{returnComponent('merchants')}</Route>
           <Route path="/:id">{returnComponent('seller')}</Route>
-          <Route path="/:id#story">{returnComponent('seller')}</Route>
+          <Route path="/:id#story">{returnComponent('seller')}</Route> */}
```

Comment:
what's this `*/`? is that leftover from commenting something out?

Reply:
this might have gotten added during some merge conflicts i was trying to resolve